### ISSUE TITLE
Missing resources found by edoweb customer

### ISSUE THREAD

**Issue by ChristophEwertowski** [hbz/lobid-resources#831]
Edoweb customers got an error because two resources are missing in lobid 2.0:
http://lobid.org/hbz01/HT019679221
http://lobid.org/hbz01/HT019679222

Both are available at lobid 1.0.

Original ticket: https://jira.hbz-nrw.de/browse/EDOZWO-789

**Comment by dr0i** [comment]
Switched the index to the new one . Both resources are now there:
http://lobid.org/resources/HT019679221
http://lobid.org/resources/HT019679222
I am investigating why they were missing in the old index. I suspect it has something to do with their creation time (Saturday).

**Comment by dr0i** [comment]
Theses resources were successfully indexed . Shortly after that the new fullindex `resources-20180506-0300` was build and updated with some files 
(`$ grep 'DE-605-aleph-update-marcxchange-'  201805* | grep  resources-2018`)
of which the newest one is:

> DE-605-aleph-update-marcxchange-20180504-20180505.tar.gz

but  it should be `DE-605-aleph-update-marcxchange-20180505-20180506.tar.gz` (where the two missing resources reside).

**Comment by dr0i** [comment]
Figured that out:
The file `toBeUpdateFilesXmlClobs_afterBasedump.txt` (comprising the list of the updates) wasn't updated itself for Api2.0. Affected were one day of missing updates of two fulldumps. The cause was a misconfiguration (I removed the daily updating of "stage" at Sunday because the new fulldump-indexing was just to be finished. Unfortunately the update of the update-list was coupled with the daily updating of stage...). There was even a note in the crontab to do this after the switch from Api1.0 to Api2.0.
Now I updated the `/home/hduser/git/hbz-aleph-dumping/bin/daily-hbz-aleph.sh` to update the update-list not only for Api1.0 but also for Api2.0.
TODO myself:
check next Monday if this is working.
